dumper/lib-dumper-spi/src/main/java/com/google/edwmigration/dumper/plugin/lib/dumper/spi/SnowflakeMetadataDumpFormat.java

```diff
@@ -100,21 +100,32 @@ enum Header {
     }
   }
 
-  interface WarehouseEventsHistoryFormat {
+  interface TableStorageMetricsFormat {
 
-    String AU_ZIP_ENTRY_NAME = "warehouse_events-au.csv";
+    String AU_ZIP_ENTRY_NAME = "table_storage_metrics.csv";
```

Comment:
For tables that exist both in Account_Usage and Information_Schema, I think we should keep the au suffix. See https://docs.snowflake.com/en/sql-reference/info-schema

Reply:
@dominik-klimczak Just found your previous comment. I think it brings value for tables/views that also exist in information_schema.